apps/condo/domains/user/schema/RegisterNewServiceUserService.test.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * Generated by `createservice user.RegisterNewServiceUserService`
+ */
+const faker = require('faker')
+const { makeLoggedInAdminClient } = require('@core/keystone/test.utils')
+const {
+    registerNewServiceUserByTestClient,
+    createTestUser,
+    createTestEmail,
+    makeLoggedInClient,
+    makeClientWithSupportUser,
+} = require('@condo/domains/user/utils/testSchema')
+const { EMAIL_ALREADY_REGISTERED_ERROR } = require('@condo/domains/user/constants/errors')
+const { REGISTER_NEW_SERVICE_USER_MUTATION } = require('@condo/domains/user/gql')
+
+describe('RegisterNewServiceUserService', () => {
```

Comment:
You usually have to implement two test sets when creating anything in condo

## 1. Access-checks tests:

These ones just check that this mutation or query can or cannot be run as a specific user type. Like: 

`can run mutation as Support` 

`cant run mutation as User`

In case when you create a new schema, you should add access checks on every CRUD operation. Like: 

`can read as user`

`cant update as user`

`can read as support`

`can update as support`

## 2. Logic test cases

These ones test logic of your creation, and should be run as they are about to run in production. 

-------

I see two sets of test cases here, but I think its better to separate them using `describe` 

It is also required that logic related test cases are like the real-world test cases. So if you are testing logic -- you should call your mutation not as admin, but as support. And after that you should try to login as created service account and try to fetch some info about yourself